.github/wiki/Home.md

```diff
@@ -0,0 +1 @@
+> *Insert information about your project here!*
```

Comment:
Should we replace this line with a the name of the application and what it does?

Reply:
Updated 🙏